Packages/Sequence-Unity/Sequence/SequenceSDK/EmbeddedWallet/SequenceWallet.cs

```diff
@@ -38,6 +38,7 @@ public Address GetWalletAddress()
         }
 
         public event Action<string> OnSignMessageComplete;
+        public event Action<string> OnSignMessageFailed;
```

Comment:
Would you mind please updating the PR description to mention that this change was included as well?